src/00_Introduction.md

```diff
@@ -1,8 +1,14 @@
 # Introduction
 
-[Gitpod](https://www.gitpod.io) is an online IDE that integrates with code hosting platforms such
-as GitHub, GitLab and Bitbucket. With a single click it creates ready-to-use developer workspaces
-with a full-featured IDE running in the cloud.
+[Gitpod](https://www.gitpod.io) is frictionless coding.
+
+Whether you just want to hack, have code to review or feel like trying something new on
+GitHub, Gitpod beams you right into a ready-to-code dev environment with a single click.
+
+Because Gitpod is based on OSS tech like VS Code, Docker and Kubernetes, it is familiar,
```

Comment:
I find mentioning Docker important because that is how users configure their workspaces. Kubernetes is transparent to users but good for name dropping.